Resources/config/services.xml

```diff
@@ -12,7 +12,6 @@
         <service id="elao.error_notifier.listener" class="%elao.error_notifier.listener.class%">
             <tag name="kernel.event_listener" event="kernel.exception" method="onKernelException" priority="0"/>
             <tag name="kernel.event_listener" event="kernel.request" method="onKernelRequest" priority="0"/>
-            <tag name="kernel.event_listener" event="console.error" method="onConsoleError" priority="0"/>
```

Comment:
But actually, I think you don't even have to care: add the two tags here. Registering a listener for an event that'll never be dispatched anymore has no drawback at all.

Reply:
Doesn't 3.4 fire both resulting in duplicate emails?

Reply:
Confirmed, if we subscribe to both on 3.4 it will trigger the exception email twice.    So I think the best solution here is to detect if the old exception class exists, if it does subscribe to that otherwise subscribe to the new event.

Reply:
You're right, indeed!